study-builder/tenants/register-user-in-pepper.js

```diff
@@ -121,7 +121,7 @@ function (user, context, callback) {
         if (
             !pepper_params.tempUserGuid &&
             !user.app_metadata.user_guid &&
-            !!user.user_metadata.temp_user_guid
+            user.user_metadata && !!user.user_metadata.temp_user_guid
```

Comment:
Bug fix to handle scenarios where has NO user_metada 
Added user.user_metadata &&  to explicitly check if user has user_metada to avoid: 
"TypeError: Cannot read property 'temp_user_guid' of undefined", 

Looks like !!user.user_metadata.temp_user_guid doesn't handle this scenario.

Reply:
That's expected if `user_metadata` is not defined. The `!!` can only coerce an `undefined` or `null` value, and  `temp_user_guid` is neither if `user_metadata` is not defined.